compiler/rustc_interface/src/passes.rs

```diff
@@ -999,7 +999,11 @@ fn analysis(tcx: TyCtxt<'_>, (): ()) -> Result<()> {
                         tcx.ensure().check_private_in_public(());
                     },
                     {
-                        sess.time("death_checking", || rustc_passes::dead::check_crate(tcx));
+                        sess.time("death_checking", || {
+                            tcx.hir().par_for_each_module(|module| {
```

Comment:
This `par_` has no effect right now because we don't build with `parallel_compiler` does it?

Reply:
Yes. When we are not in `parallel_compiler`, `for_each_module` and `par_for_each_module` are actually the same code.